db/seeds.rb

```diff
@@ -5,3 +5,45 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
```

Comment:
If you don't occupy these comments, you can delete them.

Reply:
ok